subjects/isthesquareachild/README.md

```diff
@@ -0,0 +1,48 @@
+## is-the-square-a-child
+
+### Instructions
+
+Write a function `IsTheSquareAChild` that takes a number and returns `true` if the number is a square and `false` otherwise.
```

Comment:
Use this instead...

```Write a function `IsTheSquareAChild` that takes two numbers and returns `true` if the second number is the square of the first and `false` otherwise.```

Reply:
changed